endpoint.go

```diff
@@ -890,6 +890,14 @@ func CreateOptionPortMapping(portBindings []types.PortBinding) EndpointOption {
 	}
 }
 
+// CreateOptionDNS function returns an option setter for dns entry option to
+// be passed to container Create method.
+func CreateOptionDNS(dns []string) EndpointOption {
+	return func(ep *endpoint) {
+		ep.generic[netlabel.DNSServers] = dns
+	}
+}
+
```

Comment:
@msabansal its fine to have windows network adapters to have specific dns servers. But from Docker POV, the configuration is per container and this configuration is passed to libnetwork when the sandbox is created. Shouldn't we reuse that and use it per-endpoint instead of adding a new option per endpoint ?